lib/attachProof.js

```diff
@@ -152,14 +161,29 @@ async function attachTicketServiceProof({client, ...operation} = {}) {
   return result.operation;
 }
 
-function _getInvocationTarget({operation}) {
+function _getOperationTarget({operation}) {
   if(operation.record) {
     return operation.record.id;
   }
   if(operation.recordPatch) {
-    return operation.recordPatch.target;
+    return operation.record.target;
```

Comment:
Is this change proper? Why?

Reply:
looks like this is a mistake, it should have been tested by updating a witness pool doc. Will retest with that and correct shortly. good catch,

Reply:
wow, I can't believe I missed this. It turns out the maintainer cli uses `attachInvocationProof` directly and just passes in an `invocationTarget`. So I made the changes here: https://github.com/veres-one/did-veres-one/pull/73/commits/2756f369649b97316d90324513cce9686ac0b580

This has been tested and was able to extract the invocationTarget from a recordPatch. I also found a deprecated parameter we might want to use `controller` so I added a FIXME for it. Let me know if this is good enough and thanks for the review.